recipes/DFO-LS/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "DFO-LS" %}
+{% set version = "1.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DFO-LS-{{ version }}.tar.gz
+  sha256: a6f20b9af9319a320c78f4f056d001028cea12c823edf683f4cf251abe42b311
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.11
+    - pandas >=0.17
+    - python >=3.6
+    - scipy >=0.18
+
+test:
+  imports:
+    - dfols
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/numericalalgorithmsgroup/dfols/
+  summary: A flexible derivative-free solver for (bound constrained) nonlinear least-squares minimization
+  license: GPL-3.0-only
```

Based on [upstream](https://github.com/numericalalgorithmsgroup/dfols/blob/083585973db2cc738e9d5dd3b446f76c3e8b77a7/dfols/__init__.py#L19-L20) I think this is more correct:
```suggestion
  license: GPL-3.0-or-later
```

---

Thanks for the correction, I was going by https://github.com/numericalalgorithmsgroup/dfols/blob/083585973db2cc738e9d5dd3b446f76c3e8b77a7/LICENSE.txt#L2 which didn't specify the 'or later' but the link you provide has it.